File: src/c01/3.scala
def larger_sq (x: Int, y: Int, z: Int): Int = {
  if (x < y && x < z) {
    sum_sq(y,z)
  } else if (y < x && y < z) {
    sum_sq(x,z)
  } else {
    sum_sq(x,y)
  }
}

def sum_sq (x: Int, y: Int): Int = {
  x * x + y * y
}


println(larger_sq(1,2,3))
println(larger_sq(4,2,3))
println(larger_sq(4,5,3))
println(larger_sq(4,1,2))

File: src/c01/3.py
def larger_sq(x, y, z):
    lowest = min(x,y,z)
    if lowest == x:
        return sum_sq(y,z)
    elif lowest == y:
        return sum_sq(x,z)
    else:
        return sum_sq(x,y)


def sum_sq(x, y):
    return x*x + y*y


print(larger_sq(1,2,3))
print(larger_sq(4,2,3))
print(larger_sq(4,5,3))
print(larger_sq(4,1,3))
